src/engraving/compat/midi/compatmidirender.cpp

```diff
@@ -1080,7 +1080,7 @@ std::set<size_t> CompatMidiRender::getNotesIndexesToRender(Chord* chord)
     }
 
     auto noteShouldBeRendered = [](Note* n) {
-        while (n->tieBack() && n != n->tieBack()->startNote()) {
+        while (n->tieBack() && !n->incomingPartialTie() && n != n->tieBack()->startNote()) {
```

Comment:
I've seen this check repeated several times: every time you are looking for a tie-back-but-not-incoming-partial. Perhaps this is worth adding a `tieBackNonPartial()` method?